aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -1856,22 +1870,8 @@ impl AptosSimulationVM {
             return discard_error_vm_status(err);
         };
 
-        let gas_params = match self.0 .0.get_gas_parameters(log_context) {
-            Err(err) => return discard_error_vm_status(err),
-            Ok(s) => s,
-        };
-        let storage_gas_params = match self.0 .0.get_storage_gas_parameters(log_context) {
-            Err(err) => return discard_error_vm_status(err),
-            Ok(s) => s,
-        };
-
-        let mut gas_meter =
```

Comment:
do we get the same gas meter now? e.g. this takes the storage parameters and the one on the right doesn't?

Reply:
The one on the right uses the same `storage_gas_params`, which are set when you create the meter (via `make_standard_gas_meter`)